questions.json

```
@@ -41,39 +41,39 @@
   },
   {
     "question": "OO란 CPU가 한 번에 처리 가능한 데이터의 크기를 말합니다. 예를 들어 32 bit 컴퓨터(x86)는 OO의 크기가 32bit, 64 bit 컴퓨터(x64)는 OO의 크기가 64bit입니다.",
-    "choices": [],
+    "choices": ["워드(Word)", "블록(Block)", "세그먼트(Segment)", "페이지(Page)"],
     "correct": "워드(Word)",
     "explanation": "",
     "sources": [],
     "tags": ["CS"]
   },
   {
     "question": "hash collosion을 다루는 방법의 대표적인 예시로는 Open Addressing 방식과 Separate Chaining 방식이 있습니다. Java의 HashMap은 OOOO 방식을 채택하고 있고, 이 방법은 hash collision 발생시 충돌된 버켓은 linked list로 값을 채워 넣습니다\n각 빈칸에 알맞은 키워드를 넣으세요",
```

```suggestion
    "question": "Hash Collosion을 다루는 방법의 대표적인 예시로는 Open Addressing 방식과 Separate Chaining 방식이 있습니다. Java의 HashMap은 OOOO 방식을 채택하고 있고, 이 방법은 Hash Collision 발생시 충돌된 버켓은 LinkedList 로 값을 채워 넣습니다\n각 빈칸에 알맞은 키워드를 넣으세요",
```

사소한 부분이지만 사용자에게 노출되는 부분인만큼, 전체적인 영단어 컨벤션을 고민해보는 것도 좋을 것 같아요. 일단 저는 최대한 공식 명칭을 사용하거나, 애플의 [스타일 가이드](https://support.apple.com/ko-kr/guide/applestyleguide/apsgb744e4a3/web) 따르는 편이에요. 참고만 해주세요!